x-pack/legacy/plugins/epm/server/datasources/create.ts

```diff
@@ -203,6 +203,8 @@ async function installAssets({
 
   if (datasetsRequested) {
     datasetsRequested.forEach(dataset => {
+      // add package name to dataset
+      dataset.packageName = registryPackageInfo.name;
```

Comment:
Is this still needed? It should be reported now by the registry directly.

Reply:
Just realised https://github.com/elastic/package-registry/pull/189 is not merged yet.